lib/reevoo_mark_utils.php

```diff
@@ -30,7 +30,10 @@ function getTrkref($options = array()) {
   }
 
   function getPaginationParams($options = array()) {
-    $page = self::presenceKey($_GET, 'reevoo_page', 1);
+    $page = isset($options['page'])
```

Comment:
Although thinking more about this we would need some server side changes as well for this to work, as on the server side we use "reevoo_" prefixed parameters when reloading the page after selecting a filter or a different page. So we cannot merge this change.